source/FilesServer/app/endpoints.py

```diff
@@ -15,6 +15,10 @@ async def download(file_name: str) -> RedirectResponse:
     return RedirectResponse(url=presigned_url)
 
 
-@router.post("/upload")
-async def upload(file: UploadFile | None = None):
-    return await service.s3_upload(file)
+@router.post("/upload", status_code=status.HTTP_200_OK)
+async def upload(file: UploadFile) -> None:
+    if not file:
```

Comment:
has you checked if fastapi+pydantic are checking this for us?
file is annotated as "UploadFile" not "UploadFile | None" so i think it shouldn't be None-able. if so, this if statement is redundant

Reply:
yup